packages/styles/scss/patterns/blocks/feature-card-block-large/_feature-card-block-large.scss

```diff
@@ -32,6 +32,9 @@ $fcb-large-custom-bp-nocopy: 752px;
     &__card.#{$prefix}--card--inverse .#{$prefix}--card__cta:visited {
       float: none;
       color: $inverse-link;
+      width: $layout-03;
```

Comment:
Looks good, though we might want to change this in the future if we're introducing icon tokens in Carbon? cc: @asudoh

Reply:
Agreed - Once Carbon gets a fix where token value customization works, we can start using it.